engine/apps/webhooks/presets/preset_options.py

```diff
@@ -25,6 +25,3 @@ def listen_for_webhook_save(sender: Webhook, instance: Webhook, raw: bool, *args
             WebhookPresetOptions.WEBHOOK_PRESETS[instance.preset].override_parameters_before_save(instance)
         else:
             raise NotImplementedError(f"Webhook references unknown preset implementation {instance.preset}")
-
-
-pre_save.connect(listen_for_webhook_save, Webhook)
```

Comment:
we're already connecting the `listen_for_webhook_save` function to the `pre_save` signal via the `@receiver` decorator. The [docs](https://docs.djangoproject.com/en/5.0/topics/signals/#connecting-receiver-functions) say you should use one or the other:
<img width="814" alt="Screenshot 2024-02-01 at 16 25 13" src="https://github.com/grafana/oncall/assets/9406895/46750929-f60f-4278-b65f-1a0b901ad9be">